app/src/main/java/org/fossasia/susi/ai/skills/settings/ChatSettingsFragment.kt

```diff
@@ -30,13 +29,6 @@ import org.fossasia.susi.ai.skills.settings.contract.ISettingsPresenter
 import org.fossasia.susi.ai.skills.settings.contract.ISettingsView
 import timber.log.Timber
 
-
-/**
- * The Fragment for Settings Activity
- *
- * Created by mayanktripathi on 10/07/17.
- */
```

Comment:
Unrelated change.

Reply:
well this should not be present in a file, so it is best to remove these while editing each file